src/rmodels.c

```diff
@@ -5567,6 +5567,15 @@ static Model LoadVOX(const char *fileName)
 unsigned char *m3d_loaderhook(char *fn, unsigned int *len) { return LoadFileData((const char *)fn, (int *)len); }
 void m3d_freehook(void *data) { UnloadFileData((unsigned char *)data); }
 
+// Comparison function for qsort
+static int m3d_compare_faces(const void *a, const void *b)
```

Comment:
I try to make functions as much self-contained as possible, avoiding dependencies to other functions, could it be implemented directly inside the function?